concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -64,3 +69,24 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
             parse_iso_date_to_timestamp(subtask.report_computed_task.timestamp.isoformat()),
             parse_iso_date_to_timestamp(self.report_computed_task_timestamp)
         )
+
+
+class TestIfObjectIsEnumInstance(TestCase):
```

Comment:
Please rename to `TestObjectEnumInstance`

Reply:
done